lib/cli/npm.js

```diff
@@ -54,6 +55,38 @@ export default class NPM {
     })).json.latest;
   }
 
+  async getLatestSafeUpgradeVersion (pkg, curVersion) {
+    const allVersions = (await this.exec('view', [pkg, 'versions'], {
+      json: true
+    })).json;
+    return this.getLatestSafeUpgradeFromVersions (curVersion, allVersions);
+  }
+
+  getLatestSafeUpgradeFromVersions (curVersion, allVersions) {
+    let safeUpgradeVer = null;
+    const curSemver = semver.parse(curVersion);
```

Comment:
this might throw an exception if curVersion is not a valid semver string

Reply:
it appears to return `null` in that case